public/js/reducers/pause.js

```diff
@@ -137,7 +137,9 @@ function getShouldIgnoreCaughtExceptions(state) {
 }
 
 function getFrames(state) {
-  return state.pause.get("frames") || [];
+  // frames should always be an Immutable Seq
+  // Return an empty Seq if the state is null
+  return state.pause.get("frames") || fromJS([]);
```

Comment:
Hmm I'm not sure if we do defaults like empty array in other selectors. 

What's the default otherwise, null? That might be more descriptive. 

@jlongster what do you think?

By the way, the best fix for me would be better immutable records so that we have a consistent API